gsl_programmation/templates/gsl_programmation/simulation_list.html

```diff
@@ -5,26 +5,55 @@
     {% dsfr_breadcrumb breadcrumb_dict %}
 {% endblock breadcrumb %}
 
+{% block extra_css %}
+    {{ block.super }}
+    <link rel="stylesheet" href="{% static 'css/simulation_list.css' %}">
+{% endblock extra_css %}
+
 {% block content %}
     <h1>
         {{ title }}
     </h1>
-    <ul class="no-list-style">
+    <ul class="simulation-cards-list no-list-style">
         {% for simulation in object_list %}
-            <li class="simulation-card">
-                {% dsfr_badge label=simulation.enveloppe.type %}
-                <h2>
-                    {{ simulation.title }}
-                </h2>
-                <p>
-                    {{ simulation.enveloppe.perimetre }}
-                </p>
-                <p>
-                    {% dsfr_badge label="112 projets acceptés" extra_classes="fr-badge--green-menthe" %}
-                    {% dsfr_badge label="18 projets refusés" extra_classes="fr-badge--pink-macaron" %}
-                    {% dsfr_badge label="5 projets notifiés" extra_classes="fr-badge--blue-cumulus" %}
-                </p>
-                <a href="{{ simulation.get_absolute_url }}">{{ simulation }}</a>
+            <li class="">
+                <a class="simulation-card" href="{{ simulation.get_absolute_url }}">
+                    <div class="simulation-card__actions">
+                        <span class="fr-icon-more-fill" />
+                    </div>
+                    <div class="simulation-card__content">
+                        {% dsfr_tag label=simulation.enveloppe.type extra_classes="fr-icon-money-euro-box-fill fr-icon--sm fr-tag--icon-left fr-mr-1v" %}
+
+                        <h2 class="simulation-card__title">
+                            {{ simulation.title }}
+                        </h2>
+                        <span>
+                            {{ simulation.enveloppe.perimetre }}
+                        </span>
+                    </div>
+                    <div class="simulation-card__creation-info">
+                        <p class="fr-text--sm fr-m-0">
+                            Créé par {{ simulation.created_by }}
+                            <br>
+                            Le {{ simulation.created_at|date:"j F Y" }}
+                            <br>
+                            <span class="fr-text--xs fr-m-0">
+                                Modifié le {{ simulation.updated_at|date:"j F Y" }} à {{ simulation.updated_at|date:"H\Hi" }}
```

Comment:
Si c'est une simulation, ça devrait être au féminin, non ?

```suggestion
                            Créée par {{ simulation.created_by }}
                            <br>
                            Le {{ simulation.created_at|date:"j F Y" }}
                            <br>
                            <span class="fr-text--xs fr-m-0">
                                Modifiée le {{ simulation.updated_at|date:"j F Y" }} à {{ simulation.updated_at|date:"H\Hi" }}
```

Reply:
Aïe, j'ai copié-collé sans faire attention. Bien vu !